R/plot_variance.R

```diff
@@ -0,0 +1,66 @@
+#' Plot as a heatmap the scaled between imputation variance for every cell
+#'
+#' This function plots the cell-level between imputation variance. The function
+#' scales the variances column-wise and plots the data image as a heatmap.
+#' Darker red cells indicate more variance, lighter cells indicate less variance.
+#' white cells represent observed cells or unobserved cells with zero between
+#' imputation variance.
+#'
+#' @param object A `mice` generated multiply imputed data set of class `mids`.
+#'
+#' @return An object of class `ggplot`.
+#' @importFrom dplyr mutate
```

Comment:
The other functions in `ggmice` use double colon notation to access functions from other packages. Please apply this in your proposed function as well, e.g. for the `across()` call inside `mutate()` (see below).